### ISSUE TITLE
Develop Python script to pull data from PDFs

### ISSUE THREAD

**Issue by riogringa** [riogringa/lawyersearch#4]
As we discussed, Jue is going to work on a Python script to pull data from PDFs.

[Here's the original source page for the PDFs.](http://www.justice.gov/eoir/free-legal-services-providers)

**Comment by jueyang** [comment]
After browsing the data more in-depth, I got the sense that a script might take _way_ longer and will produce results that would inevitably need manual corrections.

The unstructured data on the documents make it very difficult to automate. For example, keywords that can differentiate the content are only limited to "Esq", "Tel", "Fax", but none of them are consistent enough to generalize. The way a pdf is read is often by the area of the page -- there is no real good way to estimate an area for one lawyer service, since each cell varies in size. (If you'd like to know more, read [this](http://stackoverflow.com/questions/17217194/extracting-table-contents-from-a-collection-of-pdf-files/26110587#26110587).)

I've experimented with [PDFMiner](https://euske.github.io/pdfminer/) and [Tabula](http://tabula.technology/), without much fruition. The most usable files I found are the `txt` converted from `pdf`. However, it's not a huge improvement from the pdftable route you have been taking.

My suggestion is to manually structure the data for all states (unfortunately.) Here's the UI for pdf-->txt conversion. http://pdf2html.tabesugi.net:8080/
